example/app.js

```diff
@@ -81,6 +81,10 @@ search.addWidget(
     templates: {
       header: '<div class="panel-heading">Price ranges</div>',
       item: require('./templates/and.html')
+    },
+    transformData: function(data) {
+      data.name = data.name.replace(/(\d+) - (\d+)/, '$$$1 - $$$2').replace(/> (\d+)/, '> $$$1');
```

Comment:
Let's wait until we have the need to reuse the data after calling template to deepClone?

Template should really be the latest function call before rendering so we should be good.